imagelab/ImgProvider.java

```diff
@@ -490,5 +497,25 @@ public void showPixNew(String name) {
         dImage1.changeImage(this,"Second Pass");
         System.out.println("ImgProvider:showSlow: Second Pass");
     }
+    
+        public void removeContainer(){
+        dis.remove();
+    }
+    
```

Comment:
Each of the below getter methods are used by the Music.java method playTune() in order to get information about the pix array and DisplayImage object.